### DISCUSSION TITLE
Ovine Roadmap

### DISCUSSION THREAD

**Discussion by jinmingpang** [ovineio/ovine#54] [General]
我们将实时同步 `Ovine` 开发进度～ 
对 `Ovine` 感谢兴趣的小伙伴，欢迎在此留言讨论 ～

Ovine停更，OvineNext筹划中

**Comment by duanshuaimin** [comment]
谢谢!项目很赞,多个小项目使用中.

**Comment by z330789559** [comment]
我觉得下面这些可以更加完善项目：
1.配置登录页面，然后可以登录公用
2.移动端适配显示
3.下载项目源代码，或者打包后的静态文件
4.项目权限系统生效

**Comment by 13322211021** [comment]
你好，请问ovine有定期的升级计划吗？ 另外，因为ovine是基于amis的，如果amis升级，ovine是否会进行同步升级？
是否可以在ovine项目中单独对amis进行升级？
